static/js/tools/map/place_options.tsx

```diff
@@ -180,11 +176,14 @@ function updateEnclosedPlaceTypes(
     .then((resp) => resp.data);
   Promise.all([parentPlacePromise, placeTypePromise])
     .then(([parents, placeType]) => {
-      const selectedPlace = {
-        dcid: place.value.selectedPlace.dcid,
-        name: place.value.selectedPlace.name,
-        types: [placeType],
-      };
+      const selectedPlace =
```

Comment:
ah - i see both these calls happening. we'll need to manage state better here to avoid these duplicate calls.

add a todo? also, we should move Earth to flask to keep things consistent (or maybe even update KG with Planet..)

Reply:
added todos!